src/app/app-modules/service/service.component.html

```
@@ -1,31 +1,29 @@
 <app-header [showRoles]="false"></app-header>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)